src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/custom.py

```diff
@@ -27,6 +27,11 @@ def list_subscriptions(list_all=False): # pylint: disable=redefined-builtin
         sub['cloudName'] = sub.pop('environmentName', None)
     return [sub for sub in subscriptions if list_all or sub['cloudName'] == CLOUD.name]
 
+def show_subscription(subscription_name_or_id=None):
```

Comment:
We have an open issue on this: https://github.com/Azure/azure-cli/issues/569. The CLI convention is that arguments which accept a FOO name or ID are just called FOO, so this should just be 'subscription'. I recommend updating `set_active_subscription` as well.